.eslintrc

```diff
@@ -1,3 +1,7 @@
 {
-    "extends": "react-app"
+    "extends": "react-app",
```

Comment:
Mark from Stickler-CI here. Currently we don't support the `react-app` eslint preset. I can look into getting support for that preset added though.